.deepsource.toml

```diff
@@ -0,0 +1,10 @@
+version = 1
```

Comment:
Looks like a local file which shouldn't be included?

Reply:
This is a configuration file required to activate analysis in the repo. Apart from finding [issues](https://deepsource.io/gh/withshubh/redigo/issues/?category=recommended) in Go, You can use DeepSource to track test coverage, detect problems in Dockerfiles.

Reply:
@stevenh any update on this? :sparkling_heart:

Reply:
Hi @wricardo sorry I missed you reply there.

I'm happy merge the code fix but would need to do some more research on deepsource as looks like its triggering some false positives in that link.

For other projects I've used [golangci-lint](https://github.com/golangci/golangci-lint) which is good so will have a look at adding that here too.

Reply:
Draft of the golangci-lint is here: https://github.com/gomodule/redigo/pull/554 if you're interested in helping review as there are lots of fixes.

Reply:
@stevenh The false-positive rate of the DeepSource analyzer is less than 5%. You can integrate it to track test coverage, Detect problems in Dockerfiles, etc. in addition to detecting issues in Go. Read more about it [here](https://deepsource.io/go-static-code-analysis/).

Reply:
5% is pretty high IMO, lets get the fix merged without deepsource then we can look at that later, without blocking the fix.